main_logic/tts_client.py

```
@@ -1903,103 +1935,453 @@ def minimax_tts_worker(request_queue, response_queue, audio_api_key, voice_id, b
+                    return
+
+            if not task_started:
+                await _close_connection(expect_close=True)
+                _reset_task_state(clear_sid=True)
+                return
+
+            task_finished_event.clear()
+            try:
+                await ws.send(json.dumps({"event": "task_finish"}))
+                last_send_monotonic = None
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)